float32 axisLX
float32 axisLY
float32 axisRX
float32 axisRY

float32 trigL1
float32 trigL2
float32 trigR1
float32 trigR2

bool btnTriangle
bool btnCircle
bool btnSquare
bool btnX
